src/audio/module_adapter/module_adapter.c

```diff
@@ -44,54 +45,109 @@ struct comp_dev *module_adapter_new(const struct comp_driver *drv,
 	return module_adapter_new_ext(drv, config, spec, NULL, NULL);
 }
 
-static struct processing_module *module_adapter_mem_alloc(const struct comp_driver *drv,
-							  const struct comp_ipc_config *config)
+#if CONFIG_MM_DRV
+#define PAGE_SZ CONFIG_MM_DRV_PAGE_SIZE
+#else
+#include <platform/platform.h>
+#define PAGE_SZ HOST_PAGE_SIZE
+#endif
+
+static struct k_heap *module_adapter_dp_heap(const struct comp_ipc_config *config,
```

Comment:
Could we use a more descriptive name? Maybe something that includes create/alloc? The relation to DP is also unclear.

The `config` parameter is unused. Use `(void)config;` if we really need it.

Reply:
"dp" is because it's only used for DP. Config - yes, unused so far, but we probably will need it for heap size